spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,8 @@
+if $facts['os']['family'] == 'RedHat' {
```

Comment:
As stated elsewhere: I'd like to avoid this if possible.

Reply:
Will update this once I see how https://github.com/theforeman/puppet-dhcp/pull/167/files#diff-354f30a63fb0907d4ad57269548329e3R38 works out